README.md

```diff
@@ -43,10 +43,12 @@ jobs:
   serial: true
   plan:
   - get: resource-web-app
+  - task: build
+    file: resource-web-app/build.yml
   - put: resource-deploy-web-app
     params:
-      manifest: resource-web-app/manifest.yml
-      path: resource-web-app
```

Comment:
Ah, that makes sense. What I've done before is to move/copy the manifest into the output, so that the task is producing the entire app to push, and the `put` can just reference the manifest, which is one extra step but feels a bit cleaner.